docs/userguide/network/working-with-submariner.md

```diff
@@ -10,7 +10,7 @@ This document demonstrates how to use the `Submariner` to connect the network be
 
 ### Install Karmada control plane
 
-Following the steps [Install Karmada control plane](https://github.com/karmada-io/karmada#install-karmada-control-plane) in Quick Start, you can get a Karmada. 
+Following the steps [Install Karmada control plane](https://github.com/karmada-io/karmada#install-the-karmada-control-plane) in Quick Start, you can get a Karmada. 
```

Comment:
> https://github.com/karmada-io/karmada#install-the-karmada-control-plane

Why still link to `karmada-io/karmada` repo? I suppose we have documents on the website.

cc @Poor12

Reply:
Only check the broken link. In this case, change to `../../installation/installation.md`